README.md

```diff
@@ -1,4 +1,15 @@
-# intern2021
+# Journey
+Journey is a learning experience running on [.NET Interactive Notebooks](https://github.com/dotnet/interactive) where teachers can design a step-by-step path for students. Along the student's journey, teachers can create challenges, provide customized feedback, and define rules to continue.
```

Comment:
Is ".NET Interactive" more accurate than ".NET Interactive Notebooks"?

Reply:
"Along the student's journey" I'm not sure if this is the right description. Since this sentence is implying that student is somehow already in a journey, and during that journey, teacher does stuff. But the intent might be: the teacher creates the journey by doing said stuff

Reply:
"provide customized feedback" "personalized" might also be a good word. Rolls off the togue well thanks to alliteration

Reply:
"define rules to continue": "rules" seem to be conceptually distinct from progression within the lexicon of our product. How does "map learning progressions" sound?